ISSUE_TEMPLATE/testing.md

```diff
@@ -0,0 +1,82 @@
+- [ ] open PR against the `testing` branch on https://github.com/coreos/fedora-coreos-config
+- [ ] ideally have at least one other person check it and approve before merging
+
+# Build
+
+- [ ] Start a [pipeline build](https://jenkins-fedora-coreos.apps.ci.centos.org/job/fedora-coreos/job/fedora-coreos-fedora-coreos-pipeline/build?delay=0sec) (select `testing`, and fill in version number using the `30.YYYYMMDD.N` format)
+- [ ] Wait for the job to finish
+
+# Sanity-check the build
+
+Using the [the build browser](https://builds.coreos.fedoraproject.org/browser) for the `testing` stream:
+
+- [ ] Verify that the parent commit and version match the previous `testing` release (in the future, we'll want to integrate this check in the release job)
+- [ ] Run kola on AMI to sanity check it, e.g.:
+      `kola -p aws run --aws-ami <ami-id> --aws-region us-east-1 --parallel 10 -b fcos`
+      This will be run automatically on all builds in the future.
+
+# Sign the CHECKSUMS file for releng
+
+This is a stopgap until we do signing through fedora-messaging.
+
+- [ ] Download the `CHECKSUMS` file locally:
+      `aws s3 cp s3://fcos-builds/prod/streams/testing/builds/$VERSION/CHECKSUMS .`
+- [ ] **Confirm that the SHA256 of the `CHECKSUMS` file you just downloaded matches the one from the pipeline Jenkins log output**
+- [ ] Sign it with your key:
+      `gpg2 --output CHECKSUMS.sig --detach-sign CHECKSUMS`
+- [ ] Push your signature to the bucket:
+      `aws s3 cp --acl=public-read CHECKSUMS.sig s3://fcos-builds/prod/streams/testing/builds/$VERSION/CHECKSUMS.sig`
+
+# Signing artifacts and importing OSTree commit
+
+IMPORTANT: this is the point of no return here. Once the OSTree commit is
+imported into the unified repo, any machine that manually runs `rpm-ostree
+upgrade` will have the new update.
+
+In the future, the signing part will be integrated in the build job and the OSTree commit import will be integrated in the release job.
+
+- [ ] Open an issue on https://pagure.io/releng similar to https://pagure.io/releng/issue/8578 to ask for the artifacts to be signed and OSTree commit to be imported
+- [ ] Wait for releng to process the request
+- [ ] Verify that the sigs are present and have the right ACL, e.g.:
+      `aws s3 ls --recursive s3://fcos-builds/prod/streams/testing/builds/$VERSION/`
+      `curl -I https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/$VERSION/x86_64/fedora-coreos-$VERSION-qemu.qcow2.xz.sig`
+- [ ] optionally sanity-check that `rpm-ostree upgrade` from a machine at the
+      previous release works
+
+# Run the release job
+
+- [ ] Run the [release job](https://jenkins-fedora-coreos.apps.ci.centos.org/job/fedora-coreos/job/fedora-coreos-fedora-coreos-pipeline-release/build?delay=0sec), filling in for parameters `testing` and the new version ID
+- [ ] Wait for job to finish
+
+At this point, Cincinnati will see the new release on its next refresh and begin rolling out updates.
```

Comment:
It'll see the release, but it won't create any inbound edges to it until the stream metadata is pushed.

Reply:
Correct. The technically-correct wording for this would be something like:

> At this point, CIncinnati will see the new release on its next refresh. A corresponding release (node) will be introduced in the update graph, but no valid update paths (edges) to that will exist yet.

The `begin rolling out updates` should be moved to the next section, likely later on once we have a process for that.